sirepo/beaker_compat.py

```diff
@@ -25,7 +25,7 @@ def update_session_from_cookie_header(header):
     """
     maps = _init_maps()
     try:
-        cookie = SignedCookie(cfg.secret, input=header)
+        cookie = session.SignedCookie(cfg.secret, input=header)
```

Comment:
I think it would be clearer in this case to have `beaker.session.SignedCookie` since it is only used once, and we have a concept of `session` elsewhere in the code. In general, a fully qualified import in a single use is probably better than a partially qualified reference. It's obviously a judgement call, but I think it's worth changing in this case.